src/seeds.history.cpp

```diff
@@ -152,45 +152,15 @@ void history::trxentry(name from, name to, asset quantity) {
     return;
   }
 
-  auto from_user = users.find(from.value);
-  auto to_user = users.find(to.value);
-  
-  if (from_user == users.end() || to_user == users.end()) {
+  auto from_planted = planted.find(from.value);
+  auto to_planted = planted.find(to.value);
+
+  if (from_planted == planted.end() || to_planted == planted.end()) {
     return;
   }
 
-
-  uint64_t day = utils::get_beginning_of_day_in_seconds();
-  daily_transactions_tables transactions(get_self(), day);
-
-  uint64_t transaction_id = transactions.available_primary_key();
-  uint64_t timestamp = eosio::current_time_point().sec_since_epoch();
-
-  bool from_is_organization = from_user -> type == "organisation"_n;
-  bool to_is_organization = to_user -> type == "organisation"_n;
-
-  int64_t transactions_cap = int64_t(config_get("qev.trx.cap"_n));
-  int64_t max_transaction_points_individuals = int64_t(config_get("i.trx.max"_n));
-  int64_t max_transaction_points_organizations = int64_t(config_get("org.trx.max"_n));
-
-  double from_capped_amount = (
-    from_is_organization ? 
-    std::min(max_transaction_points_organizations, quantity.amount) : 
-    std::min(max_transaction_points_individuals, quantity.amount)
-  ) / 10000.0;
-  
-  double to_capped_amount = std::min(max_transaction_points_organizations, quantity.amount) / 10000.0;
-
-  transactions.emplace(_self, [&](auto & transaction){
-    transaction.id = transaction_id;
-    transaction.from = from;
-    transaction.to = to;
-    transaction.volume = quantity.amount;
-    transaction.qualifying_volume = std::min(transactions_cap, quantity.amount);
-    transaction.from_points = uint64_t(ceil(from_capped_amount * get_transaction_multiplier(to, from)));
-    transaction.to_points = to_is_organization ? uint64_t(ceil(to_capped_amount * get_transaction_multiplier(from, to))) : 0;
-    transaction.timestamp = timestamp;
-  });
+  bool from_is_organization = organizations.find(from.value) != organizations.end();
+  bool to_is_organization = organizations.find(to.value) != organizations.end();
```

Comment:
The weird thing is organizations keep their "fixed" planted score not in the planted table but rather in their own table ("planted")

This planted value is really the org fee for  a Ssetting up the org, I think
Anyway to figure out if a user is a Seeds user, use both the planted and org tables if one of them is != end() then it's a Seeds user.

We can make this much more lightweight by passing down the info from the token contract, which already checks those things?! Could be in the parameter for trxentry.